content/en/Platform Deep Dive/Credits/track-credits.md

```diff
@@ -1,21 +1,25 @@
 ---
 title: "Track Your Credits"
 linkTitle: "Track Your Credits"
-weight: 380
+weight: 10
+aliases: /getting-started/track-credits/
 description: >
   View and track your credits in the dashboard.
 ---
 
 {{% pageinfo %}}
-Navigate to the **Credits** page to analyze how your organization uses Cobalt [credits](https://www.cobalt.io/pentest-pricing#cobaltcredits).
+Navigate to the **Credits** page to analyze how your organization uses Cobalt [credits](/platform-deep-dive/credits/).
 {{% /pageinfo %}}
 
 Here, [Organization Owners](/getting-started/glossary/#organization-owner) and [Members](/getting-started/glossary/#organization-member) can see the following:
 
-- Current credit balance
-- Contract end date showing when your credits expire
-- History of pending and completed credit transactions
-  - You may also see your {{% ptaas-tier %}} shown as "Subscription Plan."
+- Current credit balance.
+  - Credits apply to your current contract and all relevant contracts moving forward.
```

Comment:
I _think_ you're correct, but I want to be very very careful here.

Is there a reason why you chose to add this info? (What happens if we "change our minds" and nobody updates the docs?)

Reply:
@mjang-cobalt I looked at Zendesk and initially thought that people may complain if we skip some parts. I wasn't sure about this sentence either. But you're right, we should skip this sentence.